src/frontend/packages/lib_components/src/common/Form/index.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { screen } from '@testing-library/react';
+import { render } from 'lib-tests';
+import React from 'react';
+
+import { FormHelpText } from '.';
+
+describe('<FormHelpText />', () => {
+  it('displays correctly', () => {
+    render(<FormHelpText>My text</FormHelpText>);
+
+    expect(screen.getByText('My text')).toBeInTheDocument();
```

Comment:
why not testing here the expected style to compare when the disabled props is used ?